provider/lxd/credentials_test.go

```diff
@@ -644,6 +647,61 @@ func (s *credentialsSuite) TestFinalizeCredentialNonLocal(c *gc.C) {
 	c.Assert(got, jc.DeepEquals, &expected)
 }
 
+func (s *credentialsSuite) TestFinalizeCredentialNonLocalWithCertAlreadyExists(c *gc.C) {
+	ctrl := gomock.NewController(c)
+	defer ctrl.Finish()
+
+	deps := s.createProvider(ctrl)
+
+	insecureCred := cloud.NewCredential(cloud.CertificateAuthType, map[string]string{
```

Comment:
It appears that the same insecureCred, insecureSpec, secureCred, secureSpec and params are reused in multiple tests, can we set them up to be global or shared etc.?